src/import/builders.rs

```diff
@@ -8,10 +8,12 @@ use crate::schema::helpers::int_boolean::BoolInt;
 use crate::{runtime, schema};
 use vello::peniko::{self, BlendMode, Compose, Mix};
 
+pub type LayerSetupParams = (usize, Option<BlendMode>, Option<usize>);
```

Comment:
I think this should be a named struct rather than a tuple, so it's easier to read?